lib/widget/html.dart

```diff
@@ -47,7 +47,7 @@ class HtmlState extends framework.WidgetState<EnsembleHtml> {
   Widget buildWidget(BuildContext context) {
     return Html(
       data: widget._controller.text ?? '',
-      onLinkTap: ((url, context, attributes, element) {
+      onLinkTap: ((url, attributes, element) {
```

Comment:
did this too

Reply:
@vusters I think in this we need to have attributes instead of context.
```
<<<<<<< flutter-migration
      onLinkTap: ((url, attributes, element) {
=======
      onLinkTap: ((url, context, element) {
>>>>>>> main
```